sql/2022/media/animated_gif_count.sql

```diff
@@ -0,0 +1,35 @@
+#standardSQL
+# what percent of gifs are animated?
+
+# ImageMagick reports big images as having, e.g., "1.29097M" pixels. This means ~1.2 million pixels, but BigQuery doesn't know that.
+CREATE TEMPORARY FUNCTION magickMillions(imageMagickNumberString STRING)
+RETURNS FLOAT64
+LANGUAGE js AS '''
```

Comment:
Tip: use `r'''` so that you don't need to escape regexp characters
```suggestion
LANGUAGE js AS r'''
```